src/blocks/share/test/share.cypress.js

```diff
@@ -210,6 +212,7 @@ describe( 'Test CoBlocks Share Block', function() {
 	/**
 	 * Test the coblocks share block custom classes & top/bottom spacing.
 	 */
+	// `pointer-events: none` prevents user mouse interaction (herited from a parent)
```

Comment:
Feel free to remove the comment if you are not able to reproduce the failing test.

Reply:
Unable to reproduce. Removing.